src/main/resources/application.yml

```diff
@@ -27,16 +27,28 @@ logging:
     com.netflix.eureka: ERROR
     com.netflix.discovery: ERROR
 
-spring:
-  jpa:
-    hibernate:
-      ddl-auto: update
-      format_sql: true
-  show-sql: true
+#spring:
+#  jpa:
+#    hibernate:
+#      ddl-auto: update
+#      format_sql: true
+#  show-sql: true
+
+#  h2:
+#  console:
+#    enabled: true
 
-  h2:
-  console:
-    enabled: true
+spring:
+   datasource:
+     driver-class-name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+     password: IchBinSuper!
+     url: jdbc:sqlserver://carrental.database.windows.net:1433;database=carrentalDB;encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30;
+     username: supermario
+   jpa:
+     hibernate:
+         ddl-auto: create-drop
```

Comment:
I am not sure if this configuration drops the complete database when the application is shut down. I will update this according to the config of the user-server and auth-service, as I have already tested a different value.

Reply:
i don't think that will drop the table. It will only create or drop table if the class exist or not afaik.